composer/callbacks/checkpoint_saver.py

```diff
@@ -256,7 +256,7 @@ class CheckpointSaver(Callback):  # noqa: D101
             progress). It should return ``True`` if a checkpoint should be saved given the current state and
             event.
 
-        weights_only (bool): If ``True``, save only the model weights instead of the entire training state.
+        save_model_weights_only (bool): If ``True``, save only the model weights instead of the entire training state.
```

Comment:
I know the suggestion was to defer these to later, but would it be easier to just revert these here?

`save_weights_only` or `weights_only` is used in [Keras]((https://keras.io/api/callbacks/model_checkpoint/), [PTL](https://pytorch-lightning.readthedocs.io/en/stable/api/pytorch_lightning.callbacks.ModelCheckpoint.html), etc. Any deviation from that standard should be the exception, not the default.

Reply:
changed to `save_weights_only`, like Keras does